src/atomate2/forcefields/jobs.py

```diff
@@ -7,26 +7,69 @@
 from typing import TYPE_CHECKING
 
 from jobflow import Maker, job
+from pymatgen.core.trajectory import Trajectory
 
 from atomate2.forcefields import MLFF
 from atomate2.forcefields.schemas import ForceFieldTaskDocument
-from atomate2.forcefields.utils import Relaxer, revert_default_dtype
```

Comment:
This has just been added by @janosh . I guess this should stay?

Reply:
Oh my bad, I intended to leave this in when I resolved the merge conflicts. Undoing this now

Reply:
Sorry about that - added back to the relax/static jobs and also added to the MD jobs